src/Update.cpp

```diff
@@ -1298,12 +1298,19 @@ int DoVacc(int ai, unsigned short int ts)
 void DoVaccNoDelay(int ai, unsigned short int ts)
 {
 	int x, y;
+	bool cumVG_OK = false;
 
-	if ((State.cumVG < P.VaccMaxCourses) && (!HOST_TO_BE_VACCED(ai)) && (Hosts[ai].inf >= InfStat_InfectiousAlmostSymptomatic) && (Hosts[ai].inf < InfStat_Dead_WasAsymp))
+	if ((!HOST_TO_BE_VACCED(ai)) && (Hosts[ai].inf >= InfStat_InfectiousAlmostSymptomatic) && (Hosts[ai].inf < InfStat_Dead_WasAsymp))
```

Comment:
Should the conditional on 1303 to be wrapped in a NOT - so that 1304 exits the function if one or more of the inner conditionals fails?

Reply:
Fix looks great :-)